backend/services/implementations/__tests__/testService.test.ts

```
@@ -29,18 +34,28 @@ describe("mongo testService", (): void => {
     userService.getUserById = jest.fn().mockReturnValue(mockAdmin);
     const res = await testService.createTest(mockTest);
 
-    expect(res.id).not.toBeNull();
-    expect(res).toMatchObject({
-      ...mockTest,
-      questions: res.questions,
-      admin: mockAdmin,
-      id: res.id,
-    });
+    assertResponseMatchesExpected(mockTest, res);
   });
 
   it("createTest invalid admin userId", async () => {
     await expect(async () => {
       await testService.createTest(mockTest);
     }).rejects.toThrowError(`userId ${mockTest.admin} not found`);
   });
+
+  it("getTestById", async () => {
+    userService.getUserById = jest.fn().mockReturnValue(mockAdmin);
+    const test = await testService.createTest(mockTest);
+    const res = await testService.getTestById(test.id);
+
+    expect(res.id).toEqual(test.id);
+    assertResponseMatchesExpected(mockTest, res);
+  });
+
+  it("getTestById id not found", async () => {
+    const testId = "62c248c0f79d6c3c9ebbea93";
+    expect(async () => {
+      await testService.getTestById(testId);
+    }).rejects.toThrowError(`Entity id ${testId} not found`);
```

let's throw a descriptive error message here! i.e `Test ID ${testId} not found`